specs/toggle.md

```diff
@@ -0,0 +1,92 @@
+# fast-toggle
+
+## Overview
+
+*Toggle* mimics a physical switch that allows users to turn things **on** or **off**.
```

Comment:
I think there could be a better word choice than things, maybe just "... a physical switch that can be turned **on**..."